src/main/kotlin/io.javalin/HaltException.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Javalin - https://javalin.io
+ * Copyright 2017 David Åse
```

Comment:
Stupid question: is it possible that someone is not using UTF8 in his editor and your family name is causing an issue? Could you write it as "Ase"?